File: src/targets/php/guzzle/fixtures/jsonObj-multiline-linebreaks.php
<?php
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
  'body' => '{"message":"foo\\\\nbar\\\\nbaz"}',
  'headers' => [
    'content-type' => 'application/json',
  ],
]);

echo $response->getBody();

File: src/targets/java/asynchttp/fixtures/jsonObj-multiline-linebreaks.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("POST", "https://httpbin.org/anything")
  .setHeader("content-type", "application/json")
  .setBody("{\"message\":\"foo\\\\nbar\\\\nbaz\"}")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();